rest_api/rest_api/schema.py

```diff
@@ -67,3 +78,6 @@ class QueryResponse(BaseModel):
     answers: List[Answer] = []
     documents: List[Document] = []
     debug: Optional[Dict] = Field(None, alias="_debug")
+
+
+PipelineHyperParams()
```

Comment:
This could be removed

Reply:
https://github.com/rafaljanwojcik/board-game-rules-explainer/pull/4/commits/d5847c61d41f6690162a7e5b728e447827a162f2